crates/turborepo-lib/src/cli.rs

```diff
@@ -369,6 +369,10 @@ pub struct RunArgs {
     pub tasks: Vec<String>,
     #[clap(last = true, hide = true)]
     pub pass_through_args: Vec<String>,
+
+    // Pass a string to enable posting Run Summaries to Vercel
+    #[clap(long, hide = true)]
```

Comment:
shame about it not being hidden in usage, may be worth opening an issue since I can't see one for this exact problem

Reply:
My unfamiliarity with Rust/clap confused me about where to file this issue, and I was afraid I'd be asked for reproductions that I wouldn't know how to create 😅 . Agreed we should file something though